### DISCUSSION TITLE
Koordinatenabfrage und -berechnung wie in WebGIS4 möglich?

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#86] [Q&A]
Wenn man in WebGIS4 mit dem Werkzeug "Koordinaten abfragen" einen Punkt in der Karte markiert hat und anschließend in der Maske "Werkzeug: Koordinaten abfragen" das Rechner-Symbol angeklickt hat ("Koordinaten eingeben") wurden die Projektion und die Koordinatenwerte dargestellt (Abb.1).
Beim Wechseln der Projektion wurden die Werte in die jeweilige Projektion umgerechnet (Abb.2).

In WebGIS5 werden die Werte der letzten Koordinatenabfrage nicht in das "Koordinaten eingeben"-Fenster übernommen.
Beim Wechseln der Projektion werden die Werte nicht neu berechnet.

Aus unserer Sicht sind die in WebGIS4 umgesetzten Features sehr praktisch und reduzieren die Wahrscheinlichkeit einer fehlerhaften Eingabe. Der User sieht sofort ein korrektes Beispiel.
Wie siehst du (seht ihr) das?

![koordinatenabfrage_abb1](https://github.com/gis-eni/webgis-forum/assets/100088983/5d1a67bb-8b6f-40b5-b3cf-1668aea6d65f)
Abb. 1
![koordinatenabfrage_abb2](https://github.com/gis-eni/webgis-forum/assets/100088983/41195f26-65ff-4c21-aa90-6625fb0f3688)
Abb. 2

**Comment by gis-eni** [comment]
Ich bin mir nicht ganz sicher, was hier für den Anwender die beste Lösung ist und was der genaue Anwendungsfall ist. 
Zur Zeit ist es im WebGIS 5 so, dass der letzten Werte die eingegeben werden beim nächsten Öffnen wieder drin sind. Wenn der Anwender Koordinaten eingibt und draufkommt, dass die Projektion falsch ist, kann er sie umstellen ohne das dabei die Werte geändert werden... Warum die Werte schon im Koordinaten Eingabe Dialog immer neu berechnet werden sollen weiß ich nicht.

Aber was solls, wenns im WebGIS 4 so war, wird es schon ein Sinn gemacht haben.
Im nächsten Build >=5.23.26xx ist das verhalten dann so wie im WebGIS 4. Bei öffnen wird immer der Punkt angezeigt, der zuletzt durch Klick oder Eingabe erstellt wurde.
